app/controllers/likes_controller.rb

```diff
@@ -4,6 +4,7 @@ def create
     puts @like
     flash[:notice] = @like.errors.full_messages.to_sentence unless @like.save
 
+    flash[:notice] = 'You have successfully liked this post.'
     redirect_to "http://127.0.0.1:3000/users/#{current_user.id}/posts/#{@like.post.id}"
```

Comment:
- [ ] [OPTIONAL] you do not need to add the domain you are redirecting to. you simply need to use the path you want to go to.
- [ ] [OPTIONAL] you can also use rails' URL helpers to make this redirect more succinct. 

```suggestion
    flash[:notice] = 'You have successfully liked this post.'
    redirect_to user_post_path(current_user, post)
```